js/flyouts.js

```diff
@@ -263,6 +272,7 @@
      * Close all of the flyouts unless we are clicking on a button inside of a flyout.
      */
     function handleToggleFlyoutMouseUp() {
+        console.log("flyout mouseup");
```

Comment:
Was that meant to stay in?

Reply:
Good catch. We don't have our linter setup on these files either, which adds to the pain.

It's a bit tough, being used to work in unit tested, CIed coded and coming into these.